### DISCUSSION TITLE
"Edit" no longer work...

### DISCUSSION THREAD

**Discussion by GreenLord** [kewisch/gdata-provider#319] [Support]
Just noticed today, editing of an entry, doesn't work anymore (TB and PfGC are up2date, Lubuntu 20.04.x). For testing, i aready removed/added subscribed calendars, but it didn't fixed it (happen with one of the recent updates before, but after rem/add of an sub'calendar, everything was working again).

Any suggestions?

/cheer

**Comment by geeksmith** [comment]
I have experienced the same issue recently. Clicking on a google event only gives me a summary and a link to the event. I can no longer right-click and edit. I can create events on my google calendars without issues. Running Thunderbird 91.5.0 (64-bit) and the extension update from 3 Jan, 2022.

**Comment by wfrpdm** [comment]
I also have this issue; checked permissions of the add-on but it has full access... also experimented with events created in Thunderbird vs created on phone or web portal... same result. as an FYI clicking on link at the bottom of the description launches browser thru which event may be edited, but it'd be nice to be able to edit within Tbird itself.

**Reply by wfrpdm** [reply]
just noticed this at top of calender... clicking on the edit button there opens the selected calendar event in tbird from which event may be edited... <face palm>

![Screenshot_2022-02-02_13-46-00](https://user-images.githubusercontent.com/38047319/152242931-d05b1444-c101-43aa-ad76-9873cc17de05.png)
